cdap-proto/src/main/java/co/cask/cdap/proto/profile/ProfileCreateRequest.java

```diff
@@ -33,7 +33,7 @@ public ProfileCreateRequest(String description, ProvisionerInfo provisioner) {
   }
 
   public ProvisionerInfo getProvisioner() {
-    return provisioner;
+    return new ProvisionerInfo(provisioner.getName(), provisioner.getProperties());
```

Comment:
please add a comment about why this is required.

Reply:
added